ci/build_setup.sh

```diff
@@ -61,15 +61,15 @@ ln -sf /thirdparty "${ENVOY_SRCDIR}"/ci/prebuilt
 ln -sf /thirdparty_build "${ENVOY_SRCDIR}"/ci/prebuilt
 
 # Setup Envoy consuming project.
-if [[ ! -a "${ENVOY_CONSUMER_SRCDIR}" ]]
+if [[ ! -a "${ENVOY_FILTER_EXAMPLE_SRCDIR}" ]]
 then
-  # TODO(htuch): Update to non-htuch when https://github.com/lyft/envoy/issues/404 is sorted.
-  git clone https://github.com/htuch/envoy-consumer.git "${ENVOY_CONSUMER_SRCDIR}"
+  git clone https://github.com/lyft/envoy-filter-example.git "${ENVOY_FILTER_EXAMPLE_SRCDIR}"
 fi
-cp -f "${ENVOY_SRCDIR}"/ci/WORKSPACE.consumer "${ENVOY_CONSUMER_SRCDIR}"/WORKSPACE
+cp -f "${ENVOY_SRCDIR}"/ci/WORKSPACE.filter.example "${ENVOY_FILTER_EXAMPLE_SRCDIR}"/WORKSPACE
 
-# This is the hash on https://github.com/htuch/envoy-consumer.git we pin to.
-(cd "${ENVOY_CONSUMER_SRCDIR}" && git checkout 94e11fa753a1e787c82cccaec642eda5e5b61ed8)
+# This is the hash on https://github.com/lyft/envoy-filter-example.git we pin to.
+# TODO(hennna): Point to updated hash.
```

Comment:
I mean the updated hash after https://github.com/lyft/envoy-filter-example/pull/1 is merged.